Dockerfile

```diff
@@ -60,6 +60,49 @@ RUN apt-get update \
         wget \
         xmlstarlet
 
+# dependencies for chrome headless
```

Comment:
Please add a link to where this long list of dependencies is defined. Without that knowledge we won't be able to maintain this when puppeteer suddenly stops working because it pulls down a chrome that needs something else.

Reply:
Does puppeteer have an apt package?

A lot of the dependencies also look like they're part of a desktop install. Perhaps we can change the base image instead to one that comes with these installed.

Reply:
@aslakhellesoy 
To begin with, I took inspiration from circle-ci: https://circleci.com/developer/orbs/orb/threetreeslight/puppeteer

Right now I am taking a look at the puppeteer documentation here: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-in-docker. I'll certainly submit a new proposition soon

@mpkorstanje 
The base image is ubuntu:20.04 for now. I think it already embedded a lot of desktop dependencies.
I blindly copy/paste the list from circle-ci, but actually some are already there.
Do you know other images which could fit more our needs?

Reply:
I'm afraid not.  The line from puppeteer in docker looks pretty reasonable:

```
google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 \
```